master/reference/calicoctl/commands/node/run.md

```diff
@@ -25,6 +25,7 @@ Usage:
                      [--init-system]
                      [--disable-docker-networking]
                      [--docker-networking-ifprefix=<IFPREFIX>]
+                     [--enable-libnetwork-labeling]
```

Comment:
we've referred to libnetwork as "docker networking" in the CLI flags - we should stick to that naming.